charts/spsh-client/templates/deployment.yaml

```diff
@@ -1,27 +1,49 @@
 apiVersion: apps/v1
 kind: Deployment
```

Comment:
Container Running As Root:
For security purposes, the container must be executed with a non-root user. I attempted to implement this criterion in our deployment but encountered the following errorr message
 **[emerg] mkdir() “/var/cache/nginx/client_temp” failed (13: Permission denied)** 

indicates that Nginx attempted to create a directory at /var/cache/nginx/client_temp but was unable to do so due to insufficient permissions.

My search came up with this result ,This problem typically arises in the context of Docker containers or server environments where Nginx is running under a user with restricted privileges, preventing it from writing to or creating directories in certain parts of the filesystem. I applied this solution, but my attempt was unsuccessful

https://chrisgunnelsprogramming.code.blog/2020/02/16/nginx-error-emerg-mkdir-var-cache-nginx-client_temp-failed-13-permission-denied/